federation-js/src/composition/validate/postComposition/__tests__/requiresFieldsMissingOnBase.test.ts

```diff
@@ -71,6 +73,12 @@ describe('requiresFieldsMissingOnBase', () => {
       Array [
         Object {
           "code": "REQUIRES_FIELDS_MISSING_ON_BASE",
+          "locations": Array [
```

Comment:
Should this location point to the `Product.id` field on `serviceC` in this case?

Reply:
same answer as above, for now until we have structured data I'll make this the argument node